src/components/AddToCart/styles.js

```diff
@@ -32,6 +32,7 @@ const Wrapper = styled.section`
       font-size: 0.75rem;
       color: var(--clr-white);
     }
+    z-index: -99;
```

Comment:
Remove the line: 35 and line: 27
This is will solve the issue of color selection buttons coming on top of the magnified image.